Makefile

```diff
@@ -677,8 +677,6 @@ edpm_deploy_prep: edpm_deploy_cleanup ## prepares the CR to install the data pla
 	pushd ${OPERATOR_BASE_DIR} && git clone ${GIT_CLONE_OPTS} $(if $(DATAPLANE_BRANCH),-b ${DATAPLANE_BRANCH}) ${DATAPLANE_REPO} "${OPERATOR_NAME}-operator" && popd
 	cp devsetup/edpm/services/* ${OPERATOR_BASE_DIR}/${OPERATOR_NAME}-operator/config/services
 	DEPLOY_DIR=${OPERATOR_BASE_DIR}/${OPERATOR_NAME}-operator/config/services KIND=OpenStackDataPlaneService bash scripts/gen-edpm-services-kustomize.sh
-	oc kustomize ${OPERATOR_BASE_DIR}/${OPERATOR_NAME}-operator/config/services | oc apply -f -
-	oc apply -f devsetup/edpm/config/ansible-ee-env.yaml
```

Comment:
Some Make shells still deploy stuff like `gen-edpm-nova-migration-ssh-key.sh` and `gen-ansibleee-ssh-key.sh`

Reply:
Moved to edpm_deploy step